script/build.ts

```diff
@@ -337,33 +329,25 @@ function copyDependencies() {
   mkdirSync(gitDir, { recursive: true })
   copySync(path.resolve(projectRoot, 'app/node_modules/dugite/git'), gitDir)
 
-  if (process.platform === 'win32') {
-    console.log('  Cleaning unneeded Git components…')
-    const files = [
-      'Bitbucket.Authentication.dll',
-      'GitHub.Authentication.exe',
-      'Microsoft.Alm.Authentication.dll',
-      'Microsoft.Alm.Git.dll',
-      'Microsoft.IdentityModel.Clients.ActiveDirectory.Platform.dll',
-      'Microsoft.IdentityModel.Clients.ActiveDirectory.dll',
-      'Microsoft.Vsts.Authentication.dll',
-      'git-askpass.exe',
-      'git-credential-manager.exe',
-      'WebView2Loader.dll',
-    ]
-
-    const mingwFolder = getDistArchitecture() === 'x64' ? 'mingw64' : 'mingw32'
```

Comment:
Likely me just not understanding how some of this stuff is working so mind explaining very simply. :) Why do we not unlink these now and why do we not care between mingw64 and mingw32?

Reply:
No, I should have taken the time to document this more thoroughly. So the files we're trying to unlink here don't exist in that location but rather in the bin folder (they probably did at some point though) which is the first problem. The second problems is that we don't want to delete Git Credential Manager (which these files belong to) since we're now using it.

And as for not caring about mingw64 v 32 it's because we (Desktop) don't support 32-bit Windows